### PR TITLE
Psv upload s3

### PR DESCRIPTION
**Pull request by appdoo18**
#### JIRA LINK ####

- Please include JIRA ticket link here if any
- Fixed amplify push and storage issues

#### CHANGES ####

- Please list the relevant changes in the PR
- Quick changes for amplify push errors and created a new storage bucket for documents to resolve region error issue and resolved the special character issue
giphy 1

#### DETAILS ####

- Any relevant details/references based on this changes
- https://github.com/aws-amplify/amplify-cli/issues/4094
- Used this to manually remove an index

#### MANDATORY GIF ####

PR will **NOT** be aproved if no GIF is included 😄
Include small/social giphy inside the parens **()** https://giphy.com/

### CODE REVIEW

File: install.cmd [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+set yellow=[33m
+set cyan=[36m
+set white=[37m
+
+echo.
+echo %yellow%Installing the AWS Amplify CLI...%reset%
+echo.
+
+:: Detect unsupported architecture
+for /f %%g in ('echo %PROCESSOR_ARCHITECTURE% ^| findstr 64') do (set is64Bit=%%g)
```
Comment (kaliweh) [inline install.cmd:15]: is this script for local env setup?

File: amplify/backend/backend-config.json [hunk 2]
```diff
@@ -7,6 +7,10 @@
     "bioMoleculesDB": {
       "service": "DynamoDB",
       "providerPlugin": "awscloudformation"
+    },
+    "psvpatents3": {
```
Comment (kaliweh) [inline amplify/backend/backend-config.json:11]: we will have to revert this later since this bucket will not trigger the processing

**Review by kaliweh — APPROVED** [review]
Added my comments and questions to the changes. You can go ahead for now and proceed with this but we need to revisit the bucket.

File: amplify/backend/backend-config.json (hunk 2 above)
Comment (appdoo18) [reply amplify/backend/backend-config.json:11]: Importing the storage bucket will cause a lot of headless errors, the easiest workaround is to create a new bucket. We can chat offline about how to approach a fix for this whether it be reverting or changing a number of other things

**Pull request merged**